### PR TITLE
CGE-132: Updated DB structure for further chats implementation

### PR DESCRIPTION
**Pull request by Jack11M**
CGE-132: Updated DB structure for further chats implementation

### CODE REVIEW

File: src/main/resources/liquibase/db.changelog-1.1.xml [hunk 1]
```diff
@@ -83,43 +77,105 @@
...
+            <column name="message_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="type" type="VARCHAR(7)">
+                <constraints nullable="true"/>
+            </column>
+
+            <column name="file_content" type="BLOB">
+                <constraints nullable="true"/>
```
Comment (rpkyrych) [inline src/main/resources/liquibase/db.changelog-1.1.xml:131]: Set nullable to true here and in 'type' value as well. I don't think we going to create attachment without attachment.

**Review by rpkyrych — CHANGES_REQUESTED** [review]
A small update is required.

File: src/main/resources/liquibase/db.changelog-1.1.xml (hunk 1 above)
Comment (Jack11M) [reply src/main/resources/liquibase/db.changelog-1.1.xml]: Done

**Review by rpkyrych — APPROVED** [review]
You've done fantastic job!

**Review by lubchik-14 — APPROVED** [review]
Awesome! But I ended up with a couple of questions.

File: src/main/resources/liquibase/db.changelog-1.1.xml [hunk 2]
```diff
@@ -83,43 +77,105 @@
...
+                <constraints primaryKey="true" unique="true" nullable="false"/>
+            </column>
+
+            <column name="message_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="type" type="VARCHAR(7)">
+                <constraints nullable="false"/>
+            </column>
```
[the remark this answers is not in the document]
Comment (rpkyrych) [reply src/main/resources/liquibase/db.changelog-1.1.xml:128]: On the last meeting we decided to store only a file type.

File: src/main/resources/liquibase/db.changelog-1.1.xml [hunk 3]
```diff
@@ -3,39 +3,33 @@
...
-            </column>
         </createTable>
     </changeSet>
 
-    <changeSet id="1.1.1" author="jack.petrov">
+    <changeSet id="1.1.1: Create a foreign key between 'chat' and 'advertisement' tables (many-to-one relationship)" author="jack.petrov">
+        <addForeignKeyConstraint baseColumnNames="advertisement_id"
+                                 baseTableName="chat"
+                                 constraintName="fk_chat_to_advertisement_id"
+                                 onDelete="CASCADE"
```
[the remark this answers is not in the document]
Comment (Jack11M) [reply src/main/resources/liquibase/db.changelog-1.1.xml]: Removed

**Pull request merged**